app/src/main/java/org/mozilla/focus/settings/ManualAddSearchEngineSettingsFragment.java

```diff
@@ -71,12 +74,13 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
-    private static boolean validateSearchFields(String engineName, String searchString, SharedPreferences sharedPreferences) {
-        if (sharedPreferences.getStringSet(SearchEngineManager.PREF_KEY_CUSTOM_SEARCH_ENGINES,
-                Collections.<String>emptySet()).contains(engineName)) {
-            return false;
-        }
+    private void showSnackbar(@StringRes final int errStrRes) {
+        Snackbar.make(getView(), errStrRes, Snackbar.LENGTH_SHORT).show();
+    }
```

Comment:
Unrelated to this change. But note that there's also ViewUtils.showBrandedSnackbar() that is used in other places.